ZUGFeRD/QuantityCodes.cs

```diff
@@ -168,6 +168,12 @@ public enum QuantityCodes
         /// </summary>
         KMT,
 
+        /// <summary>
+        /// Kilometer
+        /// Abkürzung: km (Rec20r16)
+        /// </summary>
+        KTM,

```

Comment:
Kilometre is defined as KMT in UNTDID 7143 according to ZUGFeRD 2.3 docs. Please remove this entry.